deploy/kube.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/kube-config"
```

Comment:
Shall we use this script to deploy heapster with different sinks? 
Ideally I want to have a go binary for deployment, but this script isn't a bad place to start.